third_party/terraform/tests/resource_compute_disk_test.go.erb

```diff
@@ -771,3 +795,35 @@ resource "google_compute_instance_group_manager" "manager" {
   wait_for_instances = true
 }`, diskName, mgrName)
 }
+
+<% unless version == 'ga' -%>
+func testAccComputeDisk_resourcePolicies(diskName, policyName string) string {
+	return fmt.Sprintf(`
+data "google_compute_image" "my_image" {
+  family  = "debian-9"
+  project = "debian-cloud"
+}
+
+resource "google_compute_resource_policy" "foo" {
+  name = "%s"
+  region = "us-central1"
+  snapshot_schedule_policy {
+    schedule {
+      daily_schedule {
+        days_in_cycle = 1
+        start_time = "04:00"
+      }
+    }
+  }
+}
+
+resource "google_compute_disk" "foobar" {
+  name = "%s"
+  image = "${data.google_compute_image.my_image.self_link}"
+  size = 50
+  type = "pd-ssd"
+  zone = "us-central1-a"
+  resource_policies = ["${google_compute_resource_policy.foo.self_link}"]
```

Comment:
Is it possible to use the [new reference syntax](https://www.terraform.io/docs/configuration/expressions.html#references-to-resource-attributes)?